package.json

```diff
@@ -7,15 +7,16 @@
     "type": "git",
     "url": "https://github.com/dai-shi/use-context-selector.git"
   },
+  "browser": "./dist/index.umd.js",
   "main": "./dist/index.js",
-  "module": "./src/index.js",
+  "module": "./dist/index/module.js",
   "types": "./src/index.d.ts",
   "files": [
     "src",
     "dist"
   ],
   "scripts": {
-    "compile": "babel src -d dist",
+    "compile": "microbundle build --name useContextSelector --no-compress",
```

Comment:
I don't think the name `useContentSelector` is very appropriate. It's like a function name (hook), but the name is for module, isn't it? Can we simply remove `--name ...` and let microbundle name it `use_content_selector`?

What's the background of specifying `--no-compress`?

Reply:
Agreed with the name, no-compress is so that the consumer of your library sees the original function name instead of mangled ones.